treeherder/webapp/api/jobs.py

```diff
@@ -375,6 +375,7 @@ def list(self, request, project):
 
         return Response(response_body)
 
+    # TODO remove
```

Comment:
Just making a comment to remind you to handle this TODO.

Reply:
I've made a note to handle it in a follow up pr.